change/@itwin-eslint-plugin-d973f288-c15d-4b29-8046-cd4e3cc61d8d.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Added support for TypeScript 5.0",
+  "packageName": "@itwin/eslint-plugin",
+  "email": "yashincontrol@gmail.com",
```

Comment:
i would recommend not exposing your personal/private email.
use your gh account's noreply email address

Reply:
ive created a separate issue to address this, https://github.com/iTwin/eslint-plugin/issues/9.
but for this pr, just change the email in here manually